config/fabric_module_set_admin_status.py

```diff
@@ -0,0 +1,89 @@
+#!/usr/bin/env python3
+#
+# Copyright (c) 2024, Nokia
+# All rights reserved.
+
+import re
+import subprocess
+import time
+from swsscommon.swsscommon import SonicV2Connector
+from sonic_py_common.logger import Logger
+from platform_ndk import nokia_common
+
+sfm_hw_slot_mapping = {
+    0: 15,
+    1: 16,
+    2: 17,
+    3: 18,
+    4: 19,
+    5: 20,
+    6: 21,
+    7: 22
+}

```

Comment:
There should not be any nokia specifiec code. Need to be moved to module.set_admin_state()